pygmt/datasets/earth_relief.py

```diff
@@ -73,12 +79,21 @@ def load_earth_relief(resolution="01d", region=None, registration=None):
     >>> grid = load_earth_relief(
     ...     "05m", region=[120, 160, 30, 60], registration="gridline"
     ... )
+    >>> # load the original land-only SRTM tiles from NASA
+    >>> grid = load_earth_relief(
+    ...     "03s",
+    ...     use_srtm=True,
+    ...     region=[135, 145, 35, 40],
```

Comment:
Each docstring serves as a PyGMT test and will be run for every commit. Use a smaller region to save testing time.
```suggestion
    ...     region=[135, 136, 35, 36],
```